.github/CODEOWNERS

```diff
@@ -106,6 +106,7 @@
 /x-pack/plugins/observability/public/pages/alerts @elastic/actionable-observability
 /x-pack/plugins/observability/public/pages/cases  @elastic/actionable-observability
 /x-pack/plugins/observability/public/pages/rules  @elastic/actionable-observability
+/x-pack/plugins/observability/public/pages/rule_details  @elastic/actionable-observability
```

Comment:
@maryam-saeidi, good catch!!
But I would suggest submitting this as a separate PR.